snippets/odoo-python-snippets.cson

```diff
@@ -44,15 +44,41 @@
 
 	'Many2one Field':
 		'prefix': 'oo_field_many2one'
-		'body': 'new_field_id = fields.Many2one(comodel_name="$1", string="$2", )'
+		"body": '''
+			Many2one(
+				"${10:res.partner}"${14:,
```

Comment:
The definition:
> A positional argument is a name that is not followed by an equal sign (=) and default value.
> A keyword argument is followed by an equal sign and an expression that gives its default value.
http://stackoverflow.com/questions/9450656/positional-argument-v-s-keyword-argument

By the way, this will work:
```python
new_field_id = fields.Many2one(string="My field", comodel_name="my.model")
```

They are not required, but if you specify a comodel which does not exist (or do not provide a comodel), you'll get a warning.